src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -1303,6 +1303,11 @@ protected static SyntaxList<TNode> RemoveRange<TNode>(SyntaxList<TNode> list, in
         public abstract SyntaxNode ThrowStatement(SyntaxNode expression = null);
 
         /// <summary>
+        /// Creates an expression that can be used to throw an exception.
+        /// </summary>
+        public abstract SyntaxNode ThrowExpression(SyntaxNode expression);
```

Comment:
>ThrowExpression [](start = 35, length = 15)

If this only exists in C#, why should we have it be part of the SyntaxGenerator class?  There are lots of constructs that are not in the SyntaxGenerators because they only exist in one language.  Do we want the new poly to be that the SyntaxGenerator contains all language constructs and the user need to know which are valid in C# and which are valid in VB?  I guess I am concerned that someone consuming this public API will not know that they shouldn't call this for VB.